File: Python_programing/practices/old_enough.py
#YS 6th Old enough

age = int(input("Tell me your age?\n"))

if age >= 18:
    print("You are old enough to vote!")
elif age >= 16:
    print("You are old enough to drive!")
elif age == 15:
    print("You are old enough to get a learner's permit!")
elif age >= 4:
    print("You are old enough to go to school!")
else:
    print("Go play outside, you iPad kid.")

File: c_programming/practices/old_enough.c
 // YS 6th old enough
#include <stdio.h>

int main(){
    int age =25;

    printf("how old are you: ");
    scanf("%d", &age);

    if(age >= 18) {
        printf("you are old enough to vote.\n");
    }else if (age >= 16) {
        printf(" you can get a drivers licence.\n");
    } else if (age == 15) {
        printf("your are old enough to get a driving permit.\n");
    }else if (age >= 5){
        printf("you are old enough to go to school\n");
    } else {
        printf("go play outside you ipad kid.\n");
    }
        return 0;
}